aten/src/ATen/native/cudnn/Conv.cpp

```diff
@@ -125,13 +125,16 @@ Tensor narrowGroup(const Tensor& t, int dim, int group_idx, int64_t groups) {
   return t.narrow(dim, group_idx * group_size, group_size);
 }
 
-bool support_channels_last(const Tensor& input, const Tensor& weight, int groups) {
+bool support_channels_last(
+    const Tensor& input,
+    const Tensor& weight,
```

Comment:
I don't see `weight` argument being used in the function

Reply:
This is left for next PR. (Should rename this function to `use_channels_last` though). I want to update the behavior:
It be decided by the memory_format of the weight instead of input. (or maybe `input.suggest_memory_format() == ChannelsLast || weight.suggest_memory_format() == ChannelsLast`)

I'm OK if we want to get rid of weight here for now / or put a comment there.